docs/advanced/IncreaseDbSize.md

```diff
@@ -0,0 +1,25 @@
+# Increase AsyncStorage Database Size on Android
+
+On Android there is a default size limit of 6MB an AsyncStorage. If you try to add more data to your AsyncStorage you will recieve an error like this: 
```

Comment:
```suggestion
Current Async Storage's size is set to 6MB. Going over this limit causes `database or disk is full` error.
```

1. Remove below error message
2. Remove `disclaimer` note and join the disclaimer content to the rest of the text above.